lib/dor/text_extraction/abbyy/ticket.rb

```diff
@@ -57,12 +65,17 @@ def pdfa_output
                         PdfUACompatible='true'>
             <OutputLocation>#{output_file_path}</OutputLocation>
             <FileExistsAction>Overwrite</FileExistsAction>
-            <NamingRule>#{druid}.&lt;Ext&gt;</NamingRule>
+            <NamingRule>#{bare_druid}.&lt;Ext&gt;</NamingRule>
           </ExportFormat>"
         end
 
```

Comment:
The change to `input_filepaths_field` is to let it handle the windows style path for the input file, instead of requiring the user of Abbyy to do it.